rest/server.js

```diff
@@ -1,46 +1,53 @@
 const express = require("express");
-const { v4: uuidv4 } = require("uuid");
 const bodyParser = require("body-parser");
 const app = express();
-const users = [];
+const { MongoClient } = require("mongodb");
+const ObjectId = require('mongodb').ObjectId; 
+const uri =
+  "mongodb+srv://admin:admin@cluster0.catjmgw.mongodb.net/?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+const database = client.db("test");
+const users = database.collection("users");
 
-// parse application/json
 app.use(bodyParser.json());
 app.use(express.static(__dirname + "/"));
 
 app.get("/", (req, res) => {
   res.sendFile(__dirname + "/index.html");
 });
 
-app.get("/users", function (req, res) {
-  res.send(users);
+app.get("/users", async function (req, res) {
+  res.send(await users.find().toArray());
 });
 
 app.post("/add", function (req, res) {
   const { name, lastName } = req.body;
-  const id = uuidv4();
-  users.push({
-    id,
-    name,
-    lastName,
-  });
-  res.send({ message: `Added user ${(id, name, lastName)}` });
+  
+  users.insertOne({ name, lastName });
+  
+  res.send({ message: `Added user: ${name} ${lastName}` });
 });
 
 app.put("/update", function (req, res) {
-  const { id, name, lastName } = req.body;
-  const i = users.findIndex((obj) => obj.id === id);
-  users[i] = { id, name, lastName };
-  res.send("Success");
+  const { _id, name, lastName } = req.body;
+  const query = {_id: ObjectId(_id)};
+
+  // const replace = {name,lastName};
+  // users.replaceOne(query, replace);
+
+  const update = {$set: {name,lastName}}
+  users.updateOne(query,update);
+
+  res.send({ message: `Updated user: ${_id}` });
 });
 
-//status i odpowiedz
 app.delete("/delete", function (req, res) {
-  const id = req.body.id;
+  const _id = req.body._id;
```

Comment:
destructure _id from req in params

e.g.

app.delete("/delete", function ({ body: { _id } }, res) {